config/constants.yml

```diff
@@ -840,7 +840,6 @@ additional_detail_question_types: {
   'Text': 'text',
   'Text Area': 'textarea',
   'Radio Button': 'radio_button',
-  'Checkbox': 'checkbox',
```

Comment:
Yeah, let's pair on this after this release. We are getting rid of it temporarily just to get this out.